addons/website_links/static/src/js/website_links_menu.js

```diff
@@ -8,9 +8,9 @@ odoo.define('website_links.website_links_menu', function (require) {
 'use strict';
 
 var sAnimations = require('website.content.snippets.animation');
+var websiteNavbarData = require('website.navbar');
 
-sAnimations.registry.websiteLinksMenu = sAnimations.Class.extend({
-	selector: '#o_website_links_share_page',
+var websiteLinksMenu = sAnimations.Class.extend({
```

Comment:
- Start the variable name with an uppercase letter since it is a Class/Widget declaration
- Once we have public widgets (12.3), you can and should indeed use them when you can, like here. In 12.1, this is is still "Animations" and they should not be used without selectors. Use a standard widget instead of an animation.

Reply:
@qsm-odoo Done :+1: